pyramda/prop.py

```diff
@@ -0,0 +1,8 @@
+from pyramda.function.curry import curry
+
+
+@curry
+def prop(x, obj):
+    if isinstance(obj, dict):
```

Comment:
I don't think this should be either calling `getitem` or `getattr`, it should only use `getitem`. Unlike javascript, in python it's not common to treat objects and their attributes as dictionaries implicitly.